frontend/model/captureLogs.js

```diff
@@ -189,5 +190,20 @@ sbp('sbp/selectors/register', {
   'appLogs/get' () { return getLogger()?.entries?.toArray() ?? [] },
   'appLogs/save' () { getLogger()?.save() },
   'appLogs/pauseCapture': captureLogsPause,
-  'appLogs/startCapture': captureLogsStart
+  'appLogs/startCapture': captureLogsStart,
+  'appLogs/logServer': process.env.NODE_ENV !== 'development' || !window.location.href.startsWith('https://gi')
```

Comment:
What does `'https://gi'` mean?

Reply:
Seems like `L:194` here is for specifically detecting the app run by `grunt dev --tunnel`. I noticed every external https URL created by it always starts with `https://gi...`.

Reply:
Yeah, I remember. 👍